src/components/Budget/AddBudgetModal/AddBudgetModal.js

```diff
@@ -85,19 +90,51 @@ const AddBudgetModal = ({ history, mutator, onClose, fund, budgetStatus, ledgerI
     [getFiscalYearOption, fund.id, fund.code, fund.name, history],
   );
 
+  if (isLoading) {
+    return (<Icon icon="spinner-ellipsis" />);
```

Comment:
`<Spinner />` ?

Reply:
I've changed to Spinner, it's not a new code though - just moved above check for `isLoading`